bliss/surveys/sdss.py

```diff
@@ -2,88 +2,97 @@
 import gzip
 import warnings
 from pathlib import Path
+from typing import List, Tuple, TypedDict
 
 import numpy as np
-import pytorch_lightning as pl
 import torch
 from astropy.io import fits
 from astropy.wcs import WCS, FITSFixedWarning
 from einops import rearrange
 from scipy.interpolate import RegularGridInterpolator
 from torch import Tensor
-from torch.utils.data import DataLoader, Dataset
+from torch.utils.data import DataLoader
 
 from bliss.catalog import FullCatalog, SourceType
+from bliss.simulator.background import ImageBackground
+from bliss.simulator.prior import ImagePrior, PriorConfig
+from bliss.simulator.psf import ImagePointSpreadFunction, PSFConfig
+from bliss.surveys.survey import Survey
 from bliss.utils.download_utils import download_file_to_dst
 
+SDSSFields = List[TypedDict("SDSSField", {"run": int, "camcol": int, "fields": List[int]})]
 
-class SloanDigitalSkySurvey(pl.LightningDataModule, Dataset):
+
+class SloanDigitalSkySurvey(Survey):
     BANDS = ("u", "g", "r", "i", "z")
+    BAND_IDXS = (0, 1, 2, 3, 4)
```

Comment:
Whats the point of `BAND_IDXS` is we already have `BANDS`? (Could you just do `range(len(BANDS))` to get the same thing?)

Reply:
Done.